ecs-init/config/common.go

```diff
@@ -99,3 +110,42 @@ func DockerUnixSocket() (string, bool) {
 	// eg: Docker daemon restart
 	return "/var/run", false
 }
+
+// Find AZ information from Metadata. If error return a blank result
+func EC2MetadataAZ () string {
```

Comment:
There's a convenience function in the SDK called [`Region`](https://github.com/aws/aws-sdk-go/blob/db3e1e27b1ace4fc57be9c5cf7cea0566bd12034/aws/ec2metadata/api.go#L114-L123) that does this for you.  Please just use `metadata.Region()` instead of duplicating the logic.